octopoes/.ci/docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
       args:
         ENVIRONMENT: dev
       context: .
-    command: pytest -x tests/integration --timeout=300
+    command: pytest tests/integration --timeout=300
```

Comment:
I think it's nicer to see whether just one tests fails or multiple because that gives a better indication of what goes wrong. If you want to stop pytest after a failing test you can always press ctrl-c.

Reply:
Agree; and we possibly save a lot of CI runtime as well

Reply:
Sounds good, I vaguely remember seeing this suite freeze from time to time back then, but that was the race condition you fixed I think 👍